### PR TITLE
Fixes cardboard tube duping

### LINKED ISSUE #28804: One package wrapper, infinite carboard tubes
Take a wrapper
Select your empty hand
Click the wrapper two times
A cardboard tube drops
Repeat infinitely
Profit

### PR DESCRIPTION
**Pull request by Cyberboss**
Fixes #28804

### CODE REVIEW

File: code/game/objects/items/stacks/stack.dm [hunk 1]
```diff
@@ -242,7 +242,7 @@
 	if(pulledby)
 		pulledby.start_pulling(S)
 	S.copy_evidences(src)
-	use(transfer)
+	use(transfer, TRUE)
```
Comment (nicbn) [inline code/game/objects/items/stacks/stack.dm]: Shouldn't it be TRUE, transfer?

File: code/game/objects/items/stacks/stack.dm [hunk 2]
```diff
@@ -291,7 +291,7 @@
 	user.put_in_hands(F)
 	add_fingerprint(user)
 	F.add_fingerprint(user)
-	use(amount)
+	use(amount, TRUE)
```
Comment (nicbn) [inline code/game/objects/items/stacks/stack.dm]: Same for here

File: code/game/objects/items/stacks/stack.dm (hunk 1 above)
Comment (Cyberboss) [inline code/game/objects/items/stacks/stack.dm]: nah, this passing a local var named transfer into the used parameter

File: code/game/objects/items/stacks/stack.dm (hunk 2 above)
Comment (Cyberboss) [inline code/game/objects/items/stacks/stack.dm]: nope

**Pull request merged**